pkgs/development/python-modules/torpy/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, cryptography
+, pytestCheckHook
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "torpy";
+  version = "1.1.6";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "torpyorg";
+    repo = "torpy";
+    rev = "v${version}";
+    sha256 = "sha256-Ni7GcpkxzAMtP4wBOFsi4KnxK+nC0XCZR/2Z/eS/C+w=";
+  };
+
+  propagatedBuildInputs = [
+    cryptography
+    requests
+   ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  checkPhase = ''
+    pytest tests --ignore tests/integration
+  '';
```

Comment:
Please use pytestCheckHook

Reply:
I used the example in https://ryantm.github.io/nixpkgs/languages-frameworks/python/ (search for "pytestCheckHook") as a template.